pkg/cmd/kind/delete/clusters/deleteclusters.go

```diff
@@ -48,7 +48,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", "", "sets kubeconfig path instead of $KUBECONFIG or $HOME/.kube/config")
-	cmd.Flags().BoolVar(&flags.All, "all", false, "delete all clusters")
+	cmd.Flags().BoolVar(&flags.All, "all", false, "delete all clusters associated with a kubeconfig")
```

Comment:
to me this reads like it only deletes clusters that are in a kubeconfig which is incorrect.

this will delete any nodes in the backend labeled as a kind cluster and then any kubeconfig entries that match the containers in the specified config.

Reply:
Okay, I have updated the description to be more specific about what is actually taking place. Let me know if it reads a little better. Thanks as always for the help.